scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -994,6 +1029,8 @@ class MicrobeEditor{
     // private auto nextMicrobeEntity;
     // private dictionary occupiedHexes;
 
+    // determine if nucleus is present in current cell
+    private bool isNucleusPresent = false;
```

Comment:
I don't think this should be a variable as keeping it up to date is a lot of work. Instead `getNucleusIsPresent` should return true if it finds the nucleus. And maybe it could be renamed `checkIsNucleusPresent`